.github/workflows/typing.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         python -m pip install -r types-requirements.txt
     - name: Run mypy
       run: |
-        mypy --disallow-untyped-defs --ignore-missing-imports -p ert_shared.models
+        mypy --disallow-untyped-defs --follow-imports=skip --ignore-missing-imports -p ert_shared.models
```

Comment:
Is this a mypy relaxation that is necessary in order to take in `ert_gui` ? But do we need it both here and in  `.mypy-strict.ini`?

Reply:
Unfortunately we need it now in both as ert_gui is leaking into `ert3` through `RunDialog`. We could create two .ini files specifying modules to ignore?